pkg/fetch/redhat/packagemanifest/packagemanifest_test.go

```diff
@@ -0,0 +1,105 @@
+package packagemanifest_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/pkg/errors"
+
+	"github.com/MaineK00n/vuls-data-update/pkg/fetch/redhat/packagemanifest"
+)
+
+func TestFetch(t *testing.T) {
+	tests := []struct {
+		name     string
+		testdata string
+		args     []string
+		hasError bool
+	}{
+		{
+			name:     "happy",
+			testdata: "testdata/fixtures",
+			args:     []string{"8", "9", "10"},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				testdata := strings.TrimPrefix(r.URL.Path, string(os.PathSeparator))
+				if _, err := os.Stat(testdata); err != nil {
+					http.NotFound(w, r)
+					return
+				}
+				http.ServeFile(w, r, testdata)
+			}))
+			defer ts.Close()
+
+			dir := t.TempDir()
+
+			err := packagemanifest.Fetch(tt.args, packagemanifest.WithBaseURL(fmt.Sprintf("%s/%s/rhel-%%s.html", ts.URL, tt.testdata)), packagemanifest.WithDir(dir))
```

Comment:
I understand that doing this doesn't make much sense, but I want the BaseURL to be as similar as possible to the production environment.
```suggestion
			err := packagemanifest.Fetch(tt.args, packagemanifest.WithBaseURL(fmt.Sprintf("%s/en/documentation/red_hat_enterprise_linux/%%s/html-single/package_manifest/index", ts.URL)), packagemanifest.WithDir(dir))
```

Reply:
fixed 5adfde3